csharp/src/elements/Geometry/Polygon.cs

```diff
@@ -69,6 +69,47 @@ public Vector3 Centroid
         /// <exception cref="System.ArgumentException">Thrown when coincident vertices are provided.</exception>
         public Polygon(IList<Vector3> vertices) : base(vertices){}
 
+        /// <summary>
+        /// Tests if the supplied 2D point is within the perimeter of this Polygon.
```

Comment:
See my note below on the use of "2D point".

Reply:
As above.

Reply:
Done.